pkg/dir.traversal.targets

```diff
@@ -2,8 +2,12 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <Import Project="..\dir.traversal.targets" />
-  
-  <PropertyGroup Condition="'$(OfficialBuildId)' != ''">
```

Comment:
You probably need to update other places that are using OfficialBuildId as well and likely need to move this property up in the dir.props.

Reply:
Thanks.  I added it in other places and moved the definition to dir.props.

Reply:
Thanks. I added it in other places and moved the definition to dir.props.

Reply:
Currently this would trigger if `BuildingAnOfficialBuildLeg=false` as well, should probably handle that case.

Reply:
Fixed.